app/views/shared/_banner_ie_not_supported.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="usa-alert usa-alert--warning browser-unsupported-banner margin-bottom-2">
+  <div class="usa-alert__body">
+  <p class="usa-alert__text"><%= t('shared.banner.ie_not_supported', date:  I18n.l(IdentityConfig.store.ie11_support_end_date, format: :event_date)) %></p>
+  </div>
+</div>
```

Comment:
Couple comments:

- I'd recommend using the `AlertComponent` for this for consistency and to simplify.
- `browser-unsupported-banner` doesn't exist

```suggestion
<%= render AlertComponent.new(type: :warning, class: 'margin-bottom-0') do %>
  <%= t('shared.banner.ie_not_supported', date:  I18n.l(IdentityConfig.store.ie11_support_end_date, format: :event_date)) %>
<% end %>
```